OneSignalExample/Assets/OneSignal/Editor/OSUnityEditorUtils.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.IO;
+
+using UnityEditor;
+using UnityEngine;
+
+ internal class OSUnityEditorUtils {
+     
+    // Searches for a list of file names accross the whole Unity project and provides
+    // the the GUID of the first found result.
+     internal static string FindFirstFileGUIDByName(String[] filenames) {
```

Comment:
fix indentation

Reply:
Fixup in 35397bace849b168395b28c5c29bd13b5d01bba7